### PR TITLE
Support notifications in manual QA testing script

### PR DESCRIPTION
**Pull request by likawind**
## Describe your changes and why you are making these changes
This PR adds simple SDK supports to connect to notification integrations. We also add manual QA test over slack channel which can be future taken by release process.

## Related issue number (if any)
ENG-2272

## Loom demo (if any)
Confirmed with QA testing in my own slack channel:
```
python3 manual_qa_tests/initialize.py --slack-token <TOKEN> --slack-channel general
```

## Checklist before requesting a review
- [ ] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [ ] I have performed a self-review of my code.
- [ ] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [ ] If this is a new feature, I have added unit tests and integration tests.
- [ ] I have run the integration tests locally and they are passing.
- [ ] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [ ] All features on the UI continue to work correctly.
- [ ] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: manual_qa_tests/README.md [hunk 1]
```diff
@@ -13,6 +13,9 @@ To run with more flexibility, configure the following commandline flags:
 * `--data-integration`: the data integration name. The data integration need to be pre-configured before running the script.
 * `--api-key`: the API key if different from `aqueduct.api_key()` by any reason.
 * `--example-notebooks`: also run all example notebooks
+* `--slack-token`: The Slack App bot token to integrate with slack notifications.
+* `--slack-channel`: The channel to send Slack notifications.
+* `--notification-level`: The notification threshold level. (e.g. 'Success' to recieve all notifications, 'Warning' to recieve for warning / failed workflows, and 'Error' for faile workflows only.)
```
Comment (kenxu95) [inline manual_qa_tests/README.md:18]: spelling: "receive" in two places.

Comment (kenxu95) [reply manual_qa_tests/README.md:18]: nit: "failed" not "faile"

**Review by kenxu95 — APPROVED** [review]
Note to update the release checklist if you haven't already :)

**Comment by likawind** [comment]
Updated!

**Pull request merged**